Cargo.toml

```diff
@@ -48,6 +48,9 @@ log = { version = "0.4.8", optional = true }
 [target.'cfg(unix)'.dependencies]
 libc = "0.2.149"
 
+[target.'cfg(target_os = "hermit")'.dependencies]
+libc = { package = "hermit-abi", version = "0.3.9" }
```

Comment:
I just realised I accept it on master, that was a mistake.

Reply:
I can rename it to `hermit_abi`.  Should we also rename `src/sys/unix` to `src/sys/hermit` because the prefix is `libc` is often used in this directory. But the code is more or less identical.

Reply:
Ah, I have an idea to solve it.